source/_components/notify.clicksend_tts.markdown

```diff
@@ -26,6 +26,7 @@ notify:
     username: CLICKSEND_USERNAME
     api_key: CLICKSEND_API_KEY
     recipient: PHONE_NO
+    caller: PHONE_NO
```

Comment:
You can remove this line, also the name variable. 
Only required items in the first example.